doc/manpages/man5/afp.conf.5.md

```diff
@@ -531,6 +531,40 @@ vol dbnest = *BOOLEAN* (default: *no*) **(G)**
 the CNID database in a folder called .AppleDB inside the volume root of
 each share.
 
+## Directory Cache Tuning
+
+dircachesize = *number* **(G)**
+
+> Maximum possible entries in the directory cache. The cache stores
+directories and files.
+> Default size is 8192, maximum size is 131072.
+
+dircache validation freq = *number* **(G)**
+
+> Directory cache validation frequency for external change detection.
+> Value of 1 means validate every access (default for backward
+compatibility), higher values validate less frequently.
+> If Netatalk is the only process accessing the volume you can safely
+set a value of 100 for maximum performance.
+
+dircache metadata window = *number* **(G)**
+
+> Time window in seconds for distinguishing metadata-only changes from
+content changes in directories.
+> Default: 300 seconds (5 minutes). Range: 60-3600 seconds.
+> If Netatalk is the only process accessing the volume you can safely
+set a value of 3600.
+
+dircache metadata threshold = *number* **(G)**
+
+> Maximum time difference in seconds between cached and current directory
+ctime to be considered a metadata-only change.
+> Default: 60 seconds (1 minute). Range: 10-1800 seconds.
+> If Netatalk is the only process accessing the volume you can safely
+set a value of 1800.
+
+**Note**: See configuration guide for more information
```

Comment:
elsewhere in the documentation we refer to this guide as the "Configuration chapter in the manual"

I think it makes it easier for readers to find if we refer to it in a consistent manner!

Reply:
Perfect, I was wondering